common/views/components/Buttons/Button/Button.js

```diff
@@ -62,7 +62,9 @@ const Button = ({
     >
       <span className="flex-inline flex--v-center">
         {icon && <Icon name={icon} />}
-        <span className="btn__text">{text}</span>
+        <span className={`btn__text ${spacing({ s: 0 }, { margin: ['top'] })}`}>
```

Comment:
Why does this have margin on the top?
Could we use `classNames({})` if we're adding more than 1 class?